.sauce/cypress.yml

```diff
@@ -21,16 +23,19 @@ docker:
 # own custom image or a different tag.
 #  image: saucelabs/stt-cypress-mocha-node:v5.6.0
 cypress:
-  version: 5.6.0
+  version: 6.6.0
   configFile: "cypress.json"  # We determine related files based on the location of the config file.
 suites:
-  - name: "saucy test"
+  - name: "saucy test in docker"
+    mode: docker
+    browser: "chrome"
+    config:
+      env:
+        hello: world
+      testFiles: [ "**/*.*" ] # Cypress native glob support.
+  - name: "saucy test in sauce"
+    mode: sauce
     browser: "chrome"
-    platformName: "Windows 10"
```

Comment:
Should't we keep a platform for `sauce` mode ?
(As it's kept optional only for not breaking API)

Reply:
ah let me update this.